tests/inputFields.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { test, expect } from '@playwright/test'
+
+test.describe('Input fields', async () => {
+	test.beforeEach(async ({ page }) => {
+		await page.goto('/')
+		await page.getByText('Pet Types').click()
+		await expect(page.getByRole('heading', { name: 'Pet Types' })).toBeVisible()
```

Comment:
While this technically works, more correct way is to assert expected text, rather than locator visibility.
`await expect(page.getByRole('heading')).toHaveText('Pet Types')`